js/templates/listingCard.html

```diff
@@ -42,7 +42,10 @@
   </div>
   <% if (ob.vendor) { %>
   <a class="userIconWrapper js-userName">
-    <div class="userIcon disc clrBr2 clrSh1"
+    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? `  📍  ${ob.vendor.location}` : ''}`;
```

Comment:
The patch for this PR is #665, I've removed the emoji there, @morebrownies chime in if you think it's important enough to put back in despite the potential issues.